src/ripple/overlay/impl/Manifest.cpp

```diff
@@ -127,28 +127,66 @@ Blob Manifest::getSignature () const
     return st.getFieldVL (sfSignature);
 }
 
-void
-ManifestCache::configValidatorKey(
-    std::string const& line, beast::Journal journal)
+bool
+ManifestCache::loadValidatorKeys(
+    Section const& keys,
+    beast::Journal journal)
 {
-    auto const words = beast::rfc2616::split(line.begin(), line.end(), ' ');
+    static boost::regex const re (
+        "[[:space:]]*"            // skip leading whitespace
+        "([[:alnum:]]+)"          // node identity
+        "(?:"                     // begin optional comment block
+        "[[:space:]]+"            // (skip all leading whitespace)
+        "(?:"                     // begin optional comment
+        "(.*[^[:space:]]+)"       // the comment
+        "[[:space:]]*"            // (skip all trailing whitespace)
+        ")?"                      // end optional comment
+        ")?"                      // end optional comment block
+    );
+
+    JLOG (journal.debug) <<
+        "Loading configured validator keys";
+
+    std::size_t count = 0;
+
+    for (auto const& line : keys.lines())
+    {
+        boost::smatch match;
 
-    if (words.size () != 2)
-        Throw<std::runtime_error> ("[validator_keys] format is `<key> <comment>");
+        if (!boost::regex_match (line, match, re))
+        {
+            JLOG (journal.error) <<
+                "Malformed entry: '" << line << "'";
+            return false;
+        }
 
-    auto const masterKey = parseBase58<PublicKey>(
-        TokenType::TOKEN_NODE_PUBLIC, words[0]);
+        auto const key = parseBase58<PublicKey>(
+            TokenType::TOKEN_NODE_PUBLIC, match[1]);
 
-    if (!masterKey)
-        Throw<std::runtime_error> ("Error decoding validator key");
+        if (!key)
+        {
+            JLOG (journal.error) <<
+                "Error decoding validator key: " << match[1];
+            return false;
+        }
 
-    if (publicKeyType(*masterKey) != KeyType::ed25519)
-        Throw<std::runtime_error> ("Validator key must use Ed25519");
+        if (publicKeyType(*key) != KeyType::ed25519)
+        {
+            JLOG (journal.error) <<
+                "Validator key not using Ed25519: " << match[1];
+        }
+
+        JLOG (journal.debug) << "Loaded key: " <<
+            toBase58(TokenType::TOKEN_NODE_PUBLIC, *key);
```

Comment:
FWIW, I think `match[1]` has a text representation of `key`, so you could use that text instead of calling `toBase58()`.  Your call.